service/core.go

```diff
@@ -103,7 +105,16 @@ func (lc *loggingCore) Logger() ulog.Log {
 	return lc.log
 }
 
+type authCore struct {
```

Comment:
nit: I don't like the term `core` -- it seems like a placeholder for something more descriptive